components/Datepicker/react.stories.js

```diff
@@ -0,0 +1,25 @@
+import { storiesOf } from '@storybook/react'
+import React, { useState } from 'react'
+// import classNames from 'classnames/bind'
+import Datepicker from './react'
+// import storyStyles from './react.stories.module.scss'
+
+// const cx = classNames.bind(storyStyles)
+
+storiesOf('Datepicker', module)
+  .add('default', () => {
+		const [effectiveDate, setDate] = useState(null)
+		const handleChange = date => setDate(date);
```

Comment:
This handleChange doesn't seem to serve any purpose. You could just use `onChange={setDate}` below.